cluster/agent/cmd/uninstall.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/appcelerator/amp/cluster/agent/pkg/docker"
+	"github.com/appcelerator/amp/cluster/agent/pkg/docker/stack"
+	"github.com/docker/docker/pkg/term"
+	"github.com/spf13/cobra"
+)
+
+func NewUninstallCommand() *cobra.Command {
+	uninstallCmd := &cobra.Command{
+		Use:   "uninstall",
+		Short: "Uninstall amp services from swarm environment",
+		RunE:  uninstall,
+	}
+	return uninstallCmd
+}
+
+func uninstall(cmd *cobra.Command, args []string) error {
+	stdin, stdout, stderr := term.StdStreams()
+	dockerCli := docker.NewDockerCli(stdin, stdout, stderr)
+
+	namespace := "amp"
+	if len(args) > 0 {
+		namespace = args[0]
+	}
+
+	opts := stack.RemoveOptions{
+		Namespaces: []string{namespace},
+	}
+
+	if err := stack.Remove(dockerCli, opts); err != nil {
```

Comment:
@ndegory Actually, the network removal is queued by the docker daemon and the network gets removed right after the stack is removed.